Assets/Scripts/Audio/FMODMissionComplete.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FMODMissionComplete : MonoBehaviour
+{
+    [FMODUnity.ParamRef]
+    public string param;
+    private bool completed;
+    private bool SeekSpeed = true;
+
+    void OnTriggerEnter(Collider other)
+    {
+        if (other.tag == "Player" == !completed)
+        {
+            FMODUnity.RuntimeManager.StudioSystem.setParameterByName(param, 1f, SeekSpeed);
+            completed = true;
+            Debug.Log("param 1f");
```

Comment:
I'd remove this and the other Debug.Log statement below if you don't need it anymore.